drape_frontend/visual_params.hpp

```diff
@@ -134,7 +134,7 @@ template <class ArrayT> typename ArrayT::value_type
 InterpolateByZoomLevels(int index, float lerpCoef, ArrayT const & values)
 {
   ASSERT_GREATER_OR_EQUAL(index, 0, ());
-  ASSERT_GREATER(values.size(), scales::UPPER_STYLE_SCALE, ());
+  ASSERT_GREATER_OR_EQUAL(values.size(), scales::UPPER_STYLE_SCALE, ());
   if (index < scales::UPPER_STYLE_SCALE)
     return values[index] + (values[index + 1] - values[index]) * lerpCoef;
   return values[scales::UPPER_STYLE_SCALE];
```

Comment:
Hm, then it shouldn't work. Will check.

Reply:
```
if (index < values.size() - 1)
    return values[index] + (values[index + 1] - values[index]) * lerpCoef;
  return values.back();
```
Assuming that the last value is prolonged for all last scales.
Or need to check all entries and modify all arrays (search for std::array<float, 20>)

Reply:
@gpesquero can you please try Viktar's fix?

Reply:
@vng what does it interpolate? is it safe to assume the value shouldn't change for last zoom levels?

In any case there should be no hardcoded zoom level counts like `std::array<float, 20>` - everything should be changed to use consts from `scales.hpp`

Reply:
> @gpesquero can you please try Viktar's fix instead of modifying an assert?

The code does not compile. I get this error: `no member named 'back' in 'df::ArrayView<float>'
  return values.back();`.

I presume that the code shall be the following...
`if (index < (int) values.size() - 1)`
`  return values[index] + (values[index + 1] - values[index]) * lerpCoef;`
`return values[values.size() - 1];`

(or the back() function shall be implemented...)

The code above does not seem to crash, but in any case  `ASSERT_GREATER(values.size(), scales::UPPER_STYLE_SCALE, ());` fires at all times, unless I change it to `ASSERT_GREATER_OR_EQUAL`